api/src/main/java/org/openmrs/module/dataentrystatistics/CalculateUserDateForObsCollectedByUser.java

```diff
@@ -0,0 +1,45 @@
+package org.openmrs.module.dataentrystatistics;
+
+import java.util.Date;
+
+public class CalculateUserDateForObsCollectedByUser {
```

Comment:
This class name should be changed to something like UserObsByDate because is essentially that what happen. getting users obs grouped by date.